creator-node/sequelize/migrations/20200922131913-post_vector_clock_db_migrations.js

```diff
@@ -13,88 +13,122 @@
 module.exports = {
   up: async (queryInterface, Sequelize) => {
     const transaction = await queryInterface.sequelize.transaction()
-    console.log('STARTING MIGRATION 20200922131913-post_vector_clock_db_migrations')
+    console.log(
+      'STARTING MIGRATION 20200922131913-post_vector_clock_db_migrations'
+    )
 
     // delete all data from DataTables where clock is still null + update remaining cnodeUsers clocks to 0
-    await queryInterface.sequelize.query(`
+    await queryInterface.sequelize.query(
+      `
       DELETE FROM "AudiusUsers" WHERE "clock" IS NULL;
       DELETE FROM "Tracks" WHERE "clock" IS NULL;
       DELETE FROM "Files" WHERE "clock" IS NULL;
       UPDATE "CNodeUsers" SET "clock" = 0 WHERE "clock" IS NULL;
-    `, { transaction })
+    `,
+      { transaction }
+    )
 
     await enforceClockNonNullConstraints(queryInterface, Sequelize, transaction)
 
     // add back in foreign key constraints from AudiusUsers and Tracks to Files
     // there are some records that are orphaned as a result of the 25000 limit, so we need to delete those records from the Tracks table
-    await queryInterface.sequelize.query(`
-      ALTER TABLE "AudiusUsers" ADD CONSTRAINT "AudiusUsers_coverArtFileUUID_fkey" FOREIGN KEY ("coverArtFileUUID") REFERENCES "Files" ("fileUUID") ON DELETE RESTRICT;
-      ALTER TABLE "AudiusUsers" ADD CONSTRAINT "AudiusUsers_metadataFileUUID_fkey" FOREIGN KEY ("metadataFileUUID") REFERENCES "Files" ("fileUUID") ON DELETE RESTRICT;
-      ALTER TABLE "AudiusUsers" ADD CONSTRAINT "AudiusUsers_profilePicFileUUID_fkey" FOREIGN KEY ("profilePicFileUUID") REFERENCES "Files" ("fileUUID") ON DELETE RESTRICT;
+    await queryInterface.sequelize.query(
+      `
       DELETE FROM "Tracks" WHERE "metadataFileUUID" IN (SELECT "metadataFileUUID" FROM "Tracks" t LEFT OUTER JOIN "Files" f ON t."metadataFileUUID" = f."fileUUID" WHERE f."fileUUID" IS NULL AND t."metadataFileUUID" IS NOT NULL);
       DELETE FROM "Tracks" WHERE "coverArtFileUUID" IN (SELECT "coverArtFileUUID" FROM "Tracks" t LEFT OUTER JOIN "Files" f ON t."coverArtFileUUID" = f."fileUUID" WHERE f."fileUUID" IS NULL AND t."coverArtFileUUID" IS NOT NULL);
-      ALTER TABLE "Tracks" ADD CONSTRAINT "Tracks_coverArtFileUUID_fkey " FOREIGN KEY ("coverArtFileUUID") REFERENCES "Files" ("fileUUID") ON DELETE RESTRICT;
-      ALTER TABLE "Tracks" ADD CONSTRAINT "Tracks_metadataFileUUID_fkey" FOREIGN KEY ("metadataFileUUID") REFERENCES "Files" ("fileUUID") ON DELETE RESTRICT;
-    `, { transaction })
```

Comment:
these lines are the only real changes in this file - everything else is formatting

Reply:
instead of deleting could we comment these out with a msg like this was previously run but taken out on X date?

Reply:
ah good idea - it's a bit tricky to comment out line by line since it's in a giant string, but I just pushed a change to comment out the old query and make the new one not include the constraints